CMakeLists.txt

```diff
@@ -411,28 +416,36 @@ endif(QUDA_MAGMA)
 
 # This selects arpack or parpack for Multi GPU
 if(QUDA_ARPACK)
+  enable_language(Fortran)
```

Comment:
duplicated. But I think for FindMPI to work properly this needs to come before FindMPI anyway

Reply:
Fixed in #820